CONTRIBUTING.md

```diff
@@ -28,9 +28,9 @@ If you feel that ink is missing a feature, we recommend that you bring it up in
 the [main repository] first, or alternatively, in the official [Discord server].
```

Comment:
Shouldn't `[Discord server]` be changed to `[discord server]`?

Same below with prettier & eslint.

Reply:
Yep, thanks. Nice work, auto-formatter…